requirements-test.txt

```diff
@@ -1,3 +1,3 @@
 coveralls
 django-extensions>=2.0.7
-openwisp-utils[qa]>=0.3.2
+openwisp-utils[qa]>=0.4.1,<0.5
```

Comment:
let's avoid putting a limit to this, the higher bound is already defined in django-ipam and is not necessary to define it here

Reply:
please update this